tests/template_generate_graphical_units_data.py

```diff
@@ -0,0 +1,11 @@
+from manim import *
```

Comment:
I really like this idea of having a template!

could you update the wiki?

Reply:
yep, I will do this!

Reply:
see here: https://github.com/ManimCommunity/manim/wiki/Testing#graphical-unit-test